apps/website-25/src/components/courses/__snapshots__/UnitLayout.test.tsx.snap

```diff
@@ -166,34 +166,9 @@ exports[`UnitLayout > renders default as expected 1`] = `
           <div
             class="unit__content flex flex-col flex-1 max-w-[728px] gap-4"
           >
-            <h2>
-              Welcome to the deep end!
-            </h2>
-            
-
-            <p>
-              This unit explores the core concepts that 
-              <em>
-                every
-              </em>
-               fish enthusiast needs to understand, from basic anatomy to the physics of water movement.
-We'll explore how fish
-[ ] communicate,
-[ ] navigate,
-[ ] and survive in their watery world
-...with special attention to the unique adaptations that make each species special.
-Through hands-on exercises and detailed study materials, you'll develop a strong foundation in 
-              <strong>
-                <em>
-                  <a
-                    href="https://en.wikipedia.org/wiki/Ichthyology"
-                  >
-                    ichthyology
-                  </a>
-                </em>
-              </strong>
-               (that's fish science for those who don't speak fluent marine biologist).
-            </p>
+            <div
```

Comment:
NB: this snapshot is less helpful now, because the rendering is async.

I think this is fine because it's covered by the other tests.